### PR TITLE
Added composer.json

### PR DESCRIPTION
**Pull request by ntstv**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "description": "A high performance PHP library for using multi curl for parallel http calls.",
+    "type": "library",
+    "license": "BSD",
+    "authors": [
+        {
+            "name": "Jaisen Mathai",
+            "email": "jaisen@jmathai.com"
+        }
+    ],
+    "minimum-stability": "dev",
```
Comment (dupuchba) [inline composer.json]: it should be nice to have php minimum version

**Comment by ntstv** [comment]
@dupuchba , I guess 5.3.0 should be enough?

**Comment by ericmmartin** [comment]
It would be great to have this available via composer! Any thoughts on merging @jmathai ?

**Comment by jmathai** [comment]
I must have forgot about this. Issue gh-22 was requesting this to be a composer package. Has anyone had a chance to test this?

**Comment by ericmmartin** [comment]
I'm not entirely sure how to test it until it is registered somewhere (I assume that is how it works).

I did notice this however:

    $ composer validate
    ./composer.json is valid, but with a few warnings
    See https://getcomposer.org/doc/04-schema.md for details on the schema
    License "BSD" is not a valid SPDX license identifier, see http://www.spdx.org/licenses/ if you use an open license.
    If the software is closed-source, you may use "proprietary" as license.

**Comment by jmathai** [comment]
@ericmmartin Thanks. I added a LICENSE file (apache) in hopes that helps.

**Comment by jmathai** [comment]
Merged in another PR (gh-23). Thanks for this.

**Pull request closed without merging**